packages/volto/src/components/manage/Contents/ContentsItem.jsx

```diff
@@ -156,14 +164,41 @@ export const ContentsItemComponent = ({
             to={`${item['@id']}${item.is_folderish ? '/contents' : ''}`}
           >
             <div className="expire-align">
-              <Icon
-                name={getContentIcon(item['@type'], item.is_folderish)}
-                size="20px"
-                className="icon-margin"
-                color="#878f93"
-                title={item['Type'] || item['@type']}
-              />{' '}
-              <span title={item.title}> {item.title}</span>
+              {item['@type'] === 'Image' ? (
```

Comment:
All of @tisto advice is quite valid, lazy loaded when needed, you don't want to load 100 images if you never even interact with them.